bundles/org.openhab.binding.velbus/src/main/java/org/openhab/binding/velbus/internal/handler/VelbusSensorHandler.java

```diff
@@ -89,12 +94,44 @@ public void handleCommand(ChannelUID channelUID, Command command) {
             byte[] packetBytes = packet.getBytes();
             velbusBridgeHandler.sendPacket(packetBytes);
         }
+
+        if (isButtonChannel(channelUID) && command instanceof StringType) {
+            StringType stringTypeCommand = (StringType) command;
+
+            if (stringTypeCommand.equals(PRESSED) || stringTypeCommand.equals(LONG_PRESSED)) {
+                VelbusButtonPacket packet = new VelbusButtonPacket(getModuleAddress().getChannelIdentifier(channelUID));
+
+                try {
+                    packet.Pressed();
+                    velbusBridgeHandler.sendPacket(packet.getBytes());
+                    Thread.sleep(20);
+
+                    if (stringTypeCommand.equals(LONG_PRESSED)) {
+                        packet.LongPressed();
+                        velbusBridgeHandler.sendPacket(packet.getBytes());
+                        Thread.sleep(20);
+                    }
+
+                    packet.Released();
+                    velbusBridgeHandler.sendPacket(packet.getBytes());
+                } catch (InterruptedException e) {
```

Comment:
This empty catch clause will probably become obsolete if you use the scheduler instead of a Thread.sleep.

Reply:
I've remove this from my code.